### PR TITLE
Removed Weekend Reading Side Column

### PR DESCRIPTION
**Pull request by tylernee**
Just deleted a few lines of code.

### CODE REVIEW

File: ubyssey/templates/homepage/base.html [hunk 1]
```diff
@@ -44,13 +44,7 @@ <h2 class="headline"><a href="{{ article.get_absolute_url }}?ref=frontpage">{{ a
                 </article>
                 {% endwith %}
             </div>
-            <div class="multi-zone cell right">
-                {% if day_of_week == 5 or day_of_week == 6 %}
-                    {{ components.multi_zone|safe }}
-                {% else %}
-                    {{ components.print_issue|safe }}
```
Comment (psiemens) [inline ubyssey/templates/homepage/base.html:51]: We actually want to keep this line:

`{{ components.print_issue|safe }}`

Sorry, should have been more clear in the issue. You can remove the `if/else` and just keep the `print_issue` line.

File: ubyssey/templates/homepage/base.html [hunk 2]
```diff
@@ -92,15 +86,8 @@ <h4 class="headline">{{ article.headline|safe }}</h4>
                     {% endfor %}
                 </ul>
             </div>
-            <div class="multi-zone cell right">
-                {% if day_of_week == 5 or day_of_week == 6 %}
-                    {{ components.print_issue|safe }}
```
Comment (psiemens) [inline ubyssey/templates/homepage/base.html:97]: Same here

File: ubyssey/templates/homepage/base.html (hunk 1 above)
Comment (tylernee) [reply ubyssey/templates/homepage/base.html:51]: Oh okay! So did you want me to add it back in or did you already?

Comment (psiemens) [reply ubyssey/templates/homepage/base.html:51]: Can you add it back? Thanks!

**Pull request closed without merging**